application/models/Campaign_model.php

```diff
@@ -1369,7 +1369,7 @@ public function archive_file($filetype, $office_id, $url)
                 echo 'Creating directory ' . $directory . PHP_EOL;
             }
 
-            mkdir($directory);
+            mkdir($directory, 0777, true);
```

Comment:
Hrmm, I think you just uncovered a bug. Dashboard jobs should be running as the same user as the app, it's probably running as root while the app is running as www-data. The mode should be something like `0755` or `0750`. Can you open an issue to track this?

Reply:
0777 was the implicit/default parameter in use before this change, but that parameter is further restricted by the umask, so it's probably OK perms-wise. Running jobs as root is a separate concern, though; will look into it.